cumulusci.yml

```diff
@@ -440,6 +460,8 @@ flows:
                 task: deploy_trial_config
                 options:
                     unmanaged: False
+            5:
```

Comment:
@jofsky Should this be part of the einstein template deploy? It seems like it's not in the right place.

Reply:
@MatthewBlanski  I have opened https://github.com/SalesforceFoundation/EDA/pull/1417 to fix everything.